src/s3poller.ts

```diff
@@ -80,14 +82,32 @@ export async function pollS3Source(
 
     // Handle case where there are common prefixes (folder structure)
     if (prefixes && prefixes.length > 0) {
-      const batchSize = 20
+      const batchSize = 100
```

Comment:
Super curious how many simultaneous connections the node server can have open to a single host and whether 100 things will actually fire off at once.